ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml

```diff
@@ -32,6 +32,18 @@ images:
   dockerfile_path: Containerfile
   from: fcos-base
   to: fcos-bastion
+- context_dir: images/bind9
+  dockerfile_path: Dockerfile
+  from: fcos-bastion
+  to: bmqe-bind9
+- context_dir: images/tang
+  dockerfile_path: Containerfile
+  from: fcos-bastion
+  to: bmqe-tang
+- context_dir: images/dhcp
+  dockerfile_path: Containerfile
+  from: fcos-bastion
```

Comment:
this should change to `centos9` (depending on openshift-qe/baremetal-qe-infra#24)